src/Lambda/Input/InvocationRequest.php

```diff
@@ -107,19 +107,12 @@ public function getQualifier(): ?string
 
     public function requestBody(): string
     {
-        $document = new \DOMDocument('1.0', 'UTF-8');
-        $document->formatOutput = true;
-
-        if (null !== $input = $this->Payload) {
-            $document->appendChild($document->createElement('Payload', $input));
-        }
-
-        return $document->saveXML();
+        return '';
     }
 
     public function requestHeaders(): array
     {
-        $headers = [];
+        $headers = ['content-type' => 'application/json'];
```

Comment:
XML?

Reply:
No. Lambda is using `rest-json`. 

Im trying to implement the "protocol feature"

Reply:
Application/json is correct